src/components/shared/__tests__/NodeCountSelector.js

```diff
@@ -0,0 +1,307 @@
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import { ThemeProvider } from 'emotion-theming';
+import React from 'react';
+import theme from 'styles/theme';
+
+import NodeCountSelector from 'shared/NodeCountSelector';
+
+function getComponent(props) {
+  return (
+    <ThemeProvider theme={theme}>
+      <NodeCountSelector {...props} />
+    </ThemeProvider>
+  );
+}
```

Comment:
What about something like `renderComponentWithTheme` instead `getComponent`? 

We could reuse this nice and handy function by storing it in  the `renderRouteWithStore.js` file that we could be renamed as `testUtils.js` or something like that